info-creator/io500_add_slurm.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Identify information from SLURM and add them into the result JSON file
+
+import sys
+import platform
+import os
+import re
+import traceback
+import subprocess
+from io500_info_editor import edit_infos, execute_download
+
+if len(sys.argv) < 2:
+  print("Synopsis: %s <SYSTEM-JSON> [<SupercomputerTypeNumber>]" % sys.argv[0])
+  sys.exit(1)
+
+json = sys.argv[1]
+supercomputerNumber = sys.argv[2] if len(sys.argv) > 2 else "0"
+
+cmd = []
+
+def infoS(key, val, unit = ""):
+  global cmd, supercomputerNumber
+  val = str(val).strip()
+  unit = unit.strip()
+  if val == None or val == "":
+    return
+  cmd.append(("Site.Supercomputer[%s]." % (supercomputerNumber) + key + "=" + val + " " + unit).strip())
```

Comment:
It the other edits, "Site.Supercomputer" has been removed. Is this a missed edit?

Reply:
The key Site.Supercomputer still exists, so it should be there.